config/application.rb

```diff
@@ -28,5 +28,7 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    config.autoload_paths << "#{Rails.root}/lib"
```

Comment:
`"#{Rails.root}/lib/vocab"` seemed much more reasonable (as it would restrict this modification to the scope of the Issue at hand).  Unfortunately, attempting to undertake this approach broke the build (please see: https://travis-ci.org/projecthydra-labs/geo_concerns/builds/91488876).

Hence, I have restored this to the path `"#{Rails.root}/lib"` (as I do not believe that any of the Rake functionality included within `"#{Rails.root}/lib/tasks"` (`"#{Rails.root}/lib/assets"` is empty) should conflict with the Modules being expose for integration with the various Rails Classes.